qutebrowser/mainwindow/tabbedbrowser.py

```diff
@@ -144,6 +143,18 @@ def widgets(self):
             w.append(self.widget(i))
         return w
 
+    @config.change_filter('tabs', 'tabbar-size')
+    @config.change_filter('fonts', 'tabbar')
+    def set_icon_size(self):
+        """Take the (fav)icon size from the config."""
+        tabbar_size = self.tabBar().height() or config.get('tabs', 'tabbar-size')
+        print(str(self.tabBar().height()) + ", " + str(config.get('tabs', 'tabbar-size')))
+        if tabbar_size == -1 or tabbar_size == 0:
+            tabbar_size = self.fontMetrics().height()
+        print(self.fontMetrics().height())
+        print(tabbar_size)
+        self.setIconSize(QSize(tabbar_size, tabbar_size))
+
```

Comment:
I did the resizing of the favicon in #791; it's a slightly different approach (you should be able to use that code with this patch as well).